
int32 joint3
int32 servo1
int32 servo2
int32 servo3
int32 centrifuga
string screenshot





